VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -265,6 +235,19 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
                 currentPlayer.rate = props.rate
             }
             
+            func changeVideoGravity(to gravity: ResizeOptions) -> String {
```

Comment:
What is the reason of passing a parameter `gravity`? If you want to make a test for it - it is reasonable - otherwise remove a parameter and use props inside.

Reply:
Well yeah, there is nothing to test actually because we don't really count anything. So I guess this func is also not really necessary.